Formula/p/python-toml.rb

```diff
@@ -18,23 +18,31 @@ class PythonToml < Formula
     sha256 cellar: :any_skip_relocation, x86_64_linux:   "a4af3afd961e422d62ca04300144dd06e722e617b8ca90d0e66177980d306c66"
   end
 
-  depends_on "python@3.11"
+  depends_on "python-setuptools" => :build
+  depends_on "python@3.11" => [:build, :test]
+  depends_on "python@3.12" => [:build, :test]
```

Comment:
Shouldn't we install it for all pythons that we have? Well, maybe from 3.9

Reply:
I think this formula is mainly intended as a library consumed by other formulae, and so my thinking is to only install for the pythons used by those dependents. That also makes it easier when we deprecate/disable `python@3.9` etc

But open to other opinions

Reply:
Gotcha, btw #149418 swaps six's 3.9 for 3.12. I can add 3.10 here too to be consistent

Reply:
Ah, nice! Let's then just add `python@3.10` in this PR I think